Resources/doc/cookbook/recipe_sortable_listing.rst

```diff
@@ -111,49 +124,82 @@ also define the sort by field to be position:
 
 .. code-block:: php
 
-   class ClientAdmin extends Admin
-   {
+    <?php
+    // src/AppBundle/Admin/ClientAdmin.php
 
-    public $last_position = 0;
+    namespace AppBundle\Admin;
 
-    private $positionService;
+    use AppBundle\Admin\BaseAdmin as Admin;
+    use Sonata\AdminBundle\Datagrid\ListMapper;
+    use Sonata\AdminBundle\Route\RouteCollection;
 
-    public function setPositionService(\Pix\SortableBehaviorBundle\Services\PositionHandler $positionHandler)
+    class ClientAdmin extends Admin
     {
-        $this->positionService = $positionHandler;
+        public $last_position = 0;
+
+        private $positionService;
+
+        protected $datagridValues = array(
+            '_page' => 1,
+            '_sort_order' => 'ASC',
+            '_sort_by' => 'position',
+        );
+
+        public function setPositionService(\Pix\SortableBehaviorBundle\Services\PositionHandler $positionHandler)
+        {
+            $this->positionService = $positionHandler;
+        }
+
+        protected function configureRoutes(RouteCollection $collection)
+        {
+            // ...
+            $collection->add('move', $this->getRouterIdParameter().'/move/{position}');
+        }
+
+        // ...
+
+        protected function configureListFields(ListMapper $listMapper)
+        {
+            $this->last_position = $this->positionService->getLastPosition($this->getRoot()->getClass());
+
+            $listMapper
+                ->addIdentifier('name')
+                ->add('enabled')
+                ->add('_action', 'actions', array(
+                    'actions' => array(
+                        'move' => array(
+                            'template' => 'AppBundle:Admin:_sort.html.twig'
+                        ),
+                    ),
+                ))
+            ;
+        }
     }
 
-    protected $datagridValues = array(
-        '_page' => 1,
-        '_sort_order' => 'ASC',
-        '_sort_by' => 'position',
-    );
+.. note::
 
-    protected function configureListFields(ListMapper $listMapper)
-    {
-        $this->last_position = $this->positionService->getLastPosition($this->getRoot()->getClass());
-        $listMapper
-            ->addIdentifier('name')
-            ->add('enabled')
-            ->add('_action', 'actions', array(
-                'actions' => array(
-                    'move' => array('template' => 'AcmeDemoBundle:Admin:_sort.html.twig'),
-                )
-            ));
-    }
+   To avoid pitfalls, do not rename ``$last_postion`` to ``$lastPostion`, as ``$last_postion`` is used directly in the template.
```

Comment:
Syntax issue: there is a single backtick as the closing of the second `$lastPostion`